packages/manager/apps/dedicated/client/app/account/contacts/service/user-contacts-service.controller.js

```diff
@@ -12,9 +12,7 @@ export default class {
         .reduce(
           (categories, category) => ({
             ...categories,
-            [category]: this.$translate.instant(
-              `account_contacts_service_category_${category}`,
-            ),
+            [category]: category,
```

Comment:
This looks like the structure could be turned to an array instead of an object but it might have a huge impact